spec.bs

```diff
@@ -3462,11 +3863,13 @@ the seller.
   original bid if the currency already matched, or a conversion provided by `scoreAd()`.
 : <dfn>ad</dfn>
 :: A [=string=]. JSON string to be passed to the scoring function.
+  TODO: It seems this and the [=generated bid/ad descriptor=] can be moved to

```

Comment:
That's great, I'd probably follow a file-up issue or make this a `class=XXX` element, so that it is tracked somewhere.